source/docs/articles/organizations/https-for-sites-using-customized-base-domains.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Enable HTTPS for Sites Using Customized Base Domains
+description: Learn how to enable HTTPS for all sites using your Organization's customized base domain.
+keywords: https development domain, base domain, change dev url, change development domain, change base domain, dev url, wildcard, cname, edge, dns, https
+---
+Enable HTTPS on all sites utilizing your organization's customized base domain by adding a [Wildcard SSL Certificate](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#wildcard) to your main site.
+
+Customized base domains on Pantheon replace `pantheon.io` as the base domain for all sites associated with your organization. This feature is available to Pantheon Partners, Strategic Partners, Enterprise accounts, Resellers, and OEM Partners. For more information, see [Customizing Your Base Domain](/docs/articles/organizations/base-domains)
+## Before You Begin
+Be sure that you have:
+
+- A customized base domain, otherwise  [request one](/docs/articles/organizations/base-domains/#request-the-base-domain) and [consider the effects](/docs/articles/organizations/base-domains/#effects-and-considerations) before proceeding
+- A Wildcard SSL Certificate from an [SSL Provider](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#ssl-providers)
+
+## Create an HTTPS endpoint
+[Add a Wildcard SSL Certificate](/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication#steps-to-enable-ssl) to your main site's Live environment. This will create an HTTPS endpoint for all sites utilizing your base domain. Sites associated with the organization (e.g. `projectsite-env.pantheonpartner.com`) and the main site (e.g. `www.pantheonpartner.com`) will be served using the same Wildcard certificate.
```

Comment:
I feel like this starts off wrong. What is my main site? Why am I adding a wildcard there? This section should be re-done and merged with the "development domains" section below. I would approach it based on whether or not the customer wants to use their primary domain name ("*.mydomain.com") or have a dedicated domain name ("*.mypantheonprojects.com") for their projects. 

That can be followed by the note that Partners shouldn't have to pay for this since they get a site for free.

Finally, we can end with DNS instructions. They should say something like "Direct the wildcard DNS to the IP address associated with the wildcard cert. This can be found within the **Domains/SSL** tab for the environment where your wildcard certificated was loaded."